server/src/test/java/au/org/aodn/ogcapi/server/core/common/RestApiTest.java

```diff
@@ -343,6 +343,23 @@ public void verifyParameterTextSearchMatch() throws IOException  {
         collections = testRestTemplate.getForEntity(getBasePath() + "/collections?filter=title='on stress coral reproduction.'", Collections.class);
         assertEquals(0, Objects.requireNonNull(collections.getBody()).getCollections().size(), "hit 0, order of words diff");
     }
+
+    @Test
+    public void verifyParameterCategoriesSearchMatch() throws IOException  {
+        super.insertJsonToElasticIndex(
+                "516811d7-cd1e-207a-e0440003ba8c79dd.json",   // Provider null
+                "7709f541-fc0c-4318-b5b9-9053aa474e0e.json"             // Provider is IMOS
+        );
+
+        ResponseEntity<Collections> collections = testRestTemplate.getForEntity(getBasePath() + "/collections?filter=categories='Saturation state of calcite in the water body'", Collections.class);
+        assertEquals(1, Objects.requireNonNull(collections.getBody()).getCollections().size(), "hit 1, only one record");
+
+        collections = testRestTemplate.getForEntity(getBasePath() + "/collections?filter=categories='Total alkalinity per unit mass of the water body, Practical salinity of the water body'", Collections.class);
```

Comment:
Please use AND / OR

Reply:
can you please confirm the path structure? like

`?filter=category='cat1' AND 'cat2'`

OR

`?filter=category='cat1' AND category='cat2'

OR something else?